flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/calcite/FlinkRelBuilderFactory.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.calcite
+
+import org.apache.calcite.plan.{Context, Contexts, RelOptCluster, RelOptSchema}
+import org.apache.calcite.rel.core.RelFactories
+import org.apache.calcite.rel.metadata.DefaultRelMetadataProvider
+import org.apache.calcite.tools.{RelBuilder, RelBuilderFactory}
+import org.apache.flink.table.plan.cost.FlinkDefaultRelMetadataProvider
+import RelFactories._
+/**
```

Comment:
Problematic comment, should change to: `The utility class is to create special [[RelBuilder]] instance which ignores merge projection operations.`

Reply:
Feel free to also apply the comment update ;)